### DISCUSSION TITLE
Do you wish that eOS had a proper upgrade functionality for Major releases like Ubuntu or Mint?

### DISCUSSION THREAD

**Discussion by amgschnappi** [elementary/triage#223] [Polls]
This (back up data, clean install next version, restore backup, reinstall apps, redo the customization, etc etc) is not my cup of tea. I would rather (back up data, upgrade, if upgrade works then continue life, if not Only then clean install next version etc etc).

**Comment by danirabbit** [comment]
Essentially a duplicate of https://github.com/orgs/elementary/discussions/120

Of course everyone wants an upgrade feature :)
